docs/quick_start.md

```diff
@@ -232,4 +232,4 @@ Tezos.wallet
 
 [boilerplate]: https://github.com/ecadlabs/taquito-boilerplate
 [smart_contract_source]: https://ide.ligolang.org/p/2sVshnZ_Aat5pIuUypIBsQ
-[smart_contract_on_better_call_dev]: https://better-call.dev/ithacanet/KT1A3dyvS4pWd9b9yLLMBKLxc6S6G5b58BsK/operations
+[smart_contract_on_better_call_dev]: https://better-call.dev/jakartanet/KT1A3dyvS4pWd9b9yLLMBKLxc6S6G5b58BsK/operations
```

Comment:
wring contract, should be KT1Hn49LVCTemdbkPpZEZnzXGm1rqtQs2HH2

Reply:
yes, fixed.